docs/model-hub/index.txt

```diff
@@ -0,0 +1,71 @@
+.. _model-hub:
+
+###########
+ Model Hub
+###########
+
+**********
+ Overview
+**********
+
+Determined's **model-hub** library makes it easy to train models from popular
+third-party libraries with a Determined cluster. With model-hub, use trusted
+implemenations of model architectures, with Determined's ability to scale
+training, track experiments, share resources and perform hyperparameter
```

Comment:
I think we should use the term distributed training instead of scale training, but not super strongly opinionated there.

Nit: Add comma `share resources, and perform...`